packages/dockest/src/run/waitForServices/index.ts

```diff
@@ -36,6 +37,11 @@ export const waitForServices = async ({
     runner.logger.debug(`${LOG_PREFIX} Initiating...`)
 
     await dockerComposeUp({ composeOpts, serviceName })
+
+    // wait until container has started before trying to resolve the container id.
+    const containerStartCheck = createContainerStartCheck({ runner })
```

Comment:
Do you think it'd be a good idea to do this kind of setup within `bootstrap` and e.g. attach the `containerStartCheck` to each `runner`?

Reply:
I just noticed that we can replace the resolveContainerId function and use the `id` property on the DockerComposeEvent instead 

https://github.com/n1ru4l/dockest/blob/4e50164cc2a56fbe7e3a407f1ef6fce572dd8bf0/packages/dockest/src/run/createDockerEventEmitter.ts#L12-L22

Reply:
The question is whether we still need a timeout for the resolveContainerId as it should definitely pop up (given that the docker-compose up command was executed before).

On CI systems it could take a long time until the image is pulled etc. 

@erikengervall What do you think?

Reply:
> The question is whether we still need a timeout for the resolveContainerId as it should definitely pop up (given that the docker-compose up command was executed before).
> 
> On CI systems it could take a long time until the image is pulled etc.
> 
> @erikengervall What do you think?

I don't mind retrieving the `containerId` from the `DockerComposeEvent`, if it's bound to be there eventually.

I do feel that timeouts are pretty useful. At least being able to configure some type of upper bound so that executions don't run forever.

Reply:
Alright I changed the code!